src/fcmp_pp/fcmp_pp_rust/fcmp++.h

```diff
@@ -0,0 +1,141 @@
+namespace fcmp_pp_rust {
+#include <cstdarg>
+#include <cstdint>
```

Comment:
`#include` after a namespace? Never seen this move. Is this the lazy way to do `using namespace` in a header? I guess this works, but it seems like something would have to go wrong somehow.